libs/client-sdk/src/CMakeLists.txt

```diff
@@ -43,14 +49,16 @@ add_library(${LIBNAME}
         serialization/json_all_flags.cpp
         flag_manager/flag_manager.cpp
         flag_manager/flag_persistence.cpp
-        bindings/c/sdk.cpp)
+        bindings/c/sdk.cpp
+)
+
 
-if (MSVC OR (NOT BUILD_SHARED_LIBS))
+if (MSVC OR LD_BUILD_STATIC_LIBS)
     target_link_libraries(${LIBNAME}
```

Comment:
We may want to note this MSVC condition in the readme.

Unless we want to figure out how to make it actually work with dynamically linked boost on windows (world of pain).